packages/sku/config/webpack/utils/index.js

```diff
@@ -1,11 +1,12 @@
 const loaders = require('./loaders');
 const { resolvePackage } = require('./resolvePackage');
+const { js, ts } = require('eslint-config-seek/extensions');
```

Comment:
Depending on a linting plugin seems dangerous.

One might change the other package only thinking you are changing what code gets linted, but suddenly find you've changed production code.

Especially given the other extensions are explicit I'd like to see this kept explicit for now.